src/app.js

```diff
@@ -38,61 +68,47 @@ export const start = async args => {
   }
 
   try {
-    const response = await getRepos(username);
-
-    const repositories = response.user.repositories.edges;
+    var userGraph = await getRepos(username);
```

Comment:
We should be using `const` here. Technically runs, but `var` working with ES6, and the introduction of `let/const` we have more explicit ways to declare variables without `var`.

Reply:
Ok thanks, I wasn’t aware of that.

Reply:
Yup! If you look up ‘Const vs let vs var’ You’ll find a bunch of good reads